bundles/org.openhab.binding.panasonictv/src/main/java/org/openhab/binding/panasonictv/internal/PanasonicTvBindingConstants.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
```

Comment:
This needs to be updated to 2022. You can do it automatically by invoking `mvn license:format` in the root dir of this repo.